apps/mobile/src/features/Settings/components/AppSettings/AppSettings.container.tsx

```diff
@@ -1,24 +1,132 @@
-import React, { useState } from 'react'
-import { useDispatch, useSelector } from 'react-redux'
-import { selectActiveSafe, setActiveSafe } from '@/src/store/activeSafeSlice'
-import { Address } from '@/src/types/address'
+import React from 'react'
+import { Linking } from 'react-native'
+import { router } from 'expo-router'
+
+import { Text, View } from 'tamagui'
 import { AppSettings } from './AppSettings'
+import { useTheme } from '@/src/theme/provider/safeTheme'
+import { SafeFontIcon as Icon } from '@/src/components/SafeFontIcon/SafeFontIcon'
+import { FloatingMenu } from '../FloatingMenu'
+import { LoadableSwitch } from '@/src/components/LoadableSwitch'
+import { useBiometrics } from '@/src/hooks/useBiometrics'
+import { capitalize } from '@/src/utils/formatters'
 
 export const AppSettingsContainer = () => {
-  const dispatch = useDispatch()
-  const activeSafe = useSelector(selectActiveSafe)
-  const [safeAddress, setSafeAddress] = useState('')
+  const { toggleBiometrics, isBiometricsEnabled, isLoading } = useBiometrics()
+  const { themePreference, setThemePreference } = useTheme()
+
+  const settingsSections = [
+    {
+      sectionName: 'Preferences',
+      items: [
+        {
+          label: 'Currency',
+          leftIcon: 'token',
+          onPress: () => console.log('currency'),
+          disabled: true,
+        },
+        {
+          label: 'Appearance',
+          leftIcon: 'appearance',
+          disabled: false,
+          rightNode: (
+            <FloatingMenu
+              themeVariant={themePreference}
+              onPressAction={({ nativeEvent }) => {
+                const mode = nativeEvent.event as 'auto' | 'dark' | 'light'
+                setThemePreference(mode)
+              }}
+              actions={[
+                {
+                  id: 'auto',
+                  title: 'Auto',
+                },
+                {
+                  id: 'dark',
+                  title: 'Dark',
+                },
+                {
+                  id: 'light',
+                  title: 'Light',
+                },
+              ]}
+            >
```

Comment:
That implementation doesn't work very well. In the designs the arrow for appeareace points down indicating it is a dropdown. In our implementation it is arrow right, which makes me think that it is a menu. I can click on the "appearance" text itself and it does nothing and that is confusing. I have to click directly on the "auto" button and then I see the dropdown. 

If hard to fix right now, open an issue and let's do it in a different sprint.